pkg/database/vercode.go

```diff
@@ -137,6 +141,10 @@ func (v *VerificationCode) FormatSymptomDate() string {
 	return v.SymptomDate.Format(project.RFC3339Date)
 }
 
+func (v *VerificationCode) IsUserReport() bool {
```

Comment:
I think it'd be better to make the consts public

Reply:
separate PR ok?